skyportal/models/scan_report/scan_report.py

```diff
@@ -0,0 +1,40 @@
+__all__ = ["ScanReport"]
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.orm import relationship
+
+from baselayer.app.models import Base
+
+
+class ScanReport(Base):
+    """A report listing saved candidates during a scan session between two dates."""
+
+    creator_id = sa.Column(
```

Comment:
I'd like us to keep using the same names as other models, so let's use `author` here